src/cdk/commands/aslVisualizationCDK.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { AslVisualization } from '../../../src/stepFunctions/commands/visualizeStateMachine/aslVisualization'
+import { getStateMachineDefinitionFromCfnTemplate, toUnescapedAslJsonString } from '../../stepFunctions/commands/visualizeStateMachine/getStateMachineDefinitionFromCfnTemplate'
+
+export class AslVisualizationCDK extends AslVisualization {
+    public readonly templatePath: string
+    public readonly cdkAppName: string
+    public readonly stateMachineName: string
+
+    public constructor(textDocument: vscode.TextDocument, templatePath: string, cdkAppName: string, stateMachineName: string) {
```

Comment:
Nit: you can declare the class variables in the constructor's params if you're not transforming them in the constructor:
```suggestion
    public constructor(
        textDocument: vscode.TextDocument,
        public readonly templatePath: string,
        public readonly cdkAppName: string,
        public readonly stateMachineName: string
    ) {
```